segment_coverage/segment_coverage/main.py

```diff
@@ -232,7 +232,7 @@ def create_coverage_page(self, quality):
 		final_path = '{}_coverage.html'.format(path_prefix)
 		common.ensure_directory(temp_path)
 		with open(temp_path, 'w') as f:
-			f.write(html)
+			common.writeall(f.write, html)
```

Comment:
Do we need the write wrapper here when we are just writing text to a normal file?

Reply:
No, but I felt it was safer to just do it everywhere.